src/view/Place.elm

```diff
@@ -0,0 +1,54 @@
+module View.Place exposing (..)
+
+import Model.Place as Place exposing (Place)
+
+import Update.Place as PlaceUpdate
+
+import Html.Events exposing (onClick, onInput)
+import Html exposing (text, div, h1, h2, p, ul, li, body, Html, a, button, Attribute, input)
+import Html.Attributes exposing (id, list, href, placeholder)
+
+view : Place -> Html PlaceUpdate.Msg
+view model =
+  div
+    [ id "locationInputs" ]
+    [ h2
+      []
+      []
+  , input
+    [ placeholder "Location Name"
+    , onInput PlaceUpdate.NameChange
+    , Html.Attributes.value model.name
+    ]
+    []
+  , input
+    [ placeholder "Country Name"
+    , onInput PlaceUpdate.CountryChange
+    , Html.Attributes.value model.country
+    ]
+    []
+  , input
+      [ placeholder "City Name"
+      , onInput PlaceUpdate.CityChange
+      , Html.Attributes.value model.city
+      ]
+      []
+  , input
+      [ placeholder "Postal Number"
+      , onInput PlaceUpdate.PostalChange
+      , Html.Attributes.value model.postal
+      ]
+      []
+  , div
+      [ id "updateLocation" ]
```

Comment:
In css, if you give an element an id, it means that element is unique on the page.  Since we're repeating these elements (as often as there are Places), we probably want to use a class here.

Also, classes should use dash-casing; `updateLocation => update-location`